parquet-column/src/main/java/org/apache/parquet/column/statistics/SizeStatistics.java

```diff
@@ -79,8 +87,12 @@ private Builder(PrimitiveType type, int maxRepetitionLevel, int maxDefinitionLev
      * @param definitionLevel definition level of the value
      */
     public void add(int repetitionLevel, int definitionLevel) {
-      repetitionLevelHistogram[repetitionLevel]++;
-      definitionLevelHistogram[definitionLevel]++;
+      if (repetitionLevelHistogram.length > 0) {
```

Comment:
do we have benchmarks for this class?  I'm curious if there is a trade-off of always incrementing here and avoiding writing at the end (I suppose JVM might be smart enough to remove the conditionals so I'm fine keeping it as is).

Reply:
I'm not a Java expert so I prefer to keep the current impl just in case.